hopsworks-web/yo/app/scripts/controllers/profileCtrl.js

```diff
@@ -112,8 +112,8 @@ angular.module('hopsWorksApp')
                           growl.success("Your password is now updated.", {title: 'Success', ttl: 5000, referenceId: 1});
                         }, function (error) {
                           self.credentialWorking = false;
-                          self.errorMsg = error.data.errorMsg;
-                          growl.error(error.data.errorMsg, {title: 'Error', ttl: 5000, referenceId: 1});
+                          self.errorMsg = error.data.usrMsg;
```

Comment:
@ErmiasG `userMsg` might be undefined here. I suggest we handle it similarly to [this](https://github.com/logicalclocks/hopsworks/blob/v0.9.0/hopsworks-web/yo/app/scripts/controllers/jobsCtrl.js#L139).